.gitignore

```diff
@@ -1,5 +1,6 @@
 # vscode
 .vscode
+.history/

```

Comment:
If this is helpful to you, I don't mind adding it, but what's it for? :)

Reply:
Oh, that's just so that files created by the vscode file history plugin don't clutter up the changed files section!